redbot/cogs/mod/kickban.py

```diff
@@ -213,11 +225,24 @@ class KickBanMixin(MixinMeta):
 
         If days is not a number, it's treated as the first word of the reason.
         Minimum 0 days, maximum 7. Defaults to 0."""
+        author = ctx.author
+        guild = ctx.guild
+        if reason == None:
```

Comment:
`None` is singleton so you should use identity check
```suggestion
        if reason is None:
```